modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/util/router.ts

```diff
@@ -125,10 +126,10 @@ export const Routes = buildRoutes({
 											ASSETS_WEB_CONTENT_OVERVIEW:
 												'/:assetId/page/:touchpoint/:title?',
 											ASSETS_WEB_CONTENT_ROUTES:
-												'/:assetId/:tabId(page|known-individuals)/:touchpoint/:title?'
-										}
-									}
-								}
+												'/:assetId/:tabId(page|known-individuals)/:touchpoint/:title?',
```

Comment:
@marcosapmf @ViniLopes20  the "afcb" format command gave me a lot of commas and double quotes throughout my code. Is this correct and expected? That's why I'm putting this PR on hold for now

Reply:
If you only run `yarn format`, then `ci:test:sf` will fail?

Reply:
I will drop this SF commit and try your suggestion